### DISCUSSION TITLE
How do you add a new user?

### DISCUSSION THREAD

**Discussion by Sdub76** [sct/overseerr#334] [Help] [answered]
I don't see how to add a new user, short of them just logging in.  How do I control which users can do this?

Also, is there a way to set per-user paths for Radarr and Sonarr?

**Comment by sct** [comment, marked as answer]
Only users who have access to your Plex server will be able to log in. It will create their account when they do so.

There is no way to set per-user paths yet.

**Reply by KrunchMuffin** [reply]
What do you mean by "access"? As in an admin? I have users that access plex, but they don't have access to the admin.

**Reply by sct** [reply]
They login to Overseerr with their personal Plex accounts.

**Reply by jrucker2004** [reply]
Are there any plans for adding the ability for users without plex accounts to log in and request items?  I have several users who don't have access to my plex server, but still request items that I hand-deliver.

**Reply by sct** [reply]
> Are there any plans for adding the ability for users without plex accounts to log in and request items? I have several users who don't have access to my plex server, but still request items that I hand-deliver.

https://github.com/sct/overseerr/issues/335

**Reply by jrucker2004** [reply]
Dang, thanks, I read through all of the issues, and somehow missed that one!

**Comment by Bergasha** [comment]


**Reply by sct** [reply]
https://github.com/sct/overseerr/wiki/Frequently-Asked-Questions#general-q3 Check here
